app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -177,6 +209,39 @@ class Scene extends React.Component {
     this.timer = setTimeout(this.addBlocks, 200);
   };
 
+  addToOverlay(body) {
+    if (!body || !body.render.sprite.texture || body.onOverlay) {
+      return;
+    }
+    const ctx = this.overlay.current.getContext('2d');
+    const { texture } = body.render.sprite;
+    const { sprite } = body.render;
+    ctx.translate(body.position.x, body.position.y);
+    ctx.rotate(body.angle);
+
+    const img = this.getImageFromCache(texture);
```

Comment:
I'm not following the value of this. Module exports are already cached. What is the connection between `image` and `texture`?

Reply:
It's to try and avoid `image = new Image();` which seemed to be very expensive. Texture is pulled from the body object and contains the filepath for the image.